admin_guide/manage_nodes.adoc

```diff
@@ -734,10 +734,10 @@ maximums] page for the maximum supported limits for each version of
 {product-title}.
 ====
 
-In the *_/etc/origin/node/node-config.yaml_* file, two parameters control the
-maximum number of pods that can be scheduled to a node: `pods-per-core` and
-`max-pods`. When both options are in use, the lower of the two limits the number
-of pods on a node. Exceeding these values can result in:
+In the *_/etc/origin/node/node-config.yaml_* file, one parameter controls the
```

Comment:
I believe we treat file paths as a literal. Please use back ticks instead of bold.

Reply:
@mjpytlak this is for 3.11. OCP 3 docs had a different style than OCP 4 docs.

Reply:
@mjpytlak the original doc:https://docs.openshift.com/container-platform/3.11/admin_guide/manage_nodes.html#admin-guide-max-pods-per-node has the file path as bold and not a literal.